modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -218,6 +218,11 @@ of {product-title} will support defining multiple compute pools during
 installation. Only one control plane pool is used.
 <3> Whether to enable or disable simultaneous multithreading (SMT), or `hyperthreading`. By default, SMT is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable SMT, you must disable it in all cluster machines; this includes both control plane and compute machines.
 +
+[NOTE]
+====
+If you enable SMT, you must ensure that SMT is enabled in your system's BIOS/UEFI configuration.
```

Comment:
Not critical, but I might specify "If you enable SMT in the `install-config.yaml` file, you must.." 

I just had to read it twice because it says "if you enable SMT, you must ensure that it is enabled..." which sounded weird until I realized you were talking about two different places.

Reply:
Actually I just read the paragraph above, which says that SMT is enabled by default. So maybe the "If you enable SMT" isn't quite accurate, because it wasn't necessarily them that enabled it, if it's done by default? (If that makes sense ;)  )

I was also wondering whether it could be worked into the above paragraph instead of a NOTE. Just because there is a sentence saying about something that you have to do if you disable it. But it's totally fine to keep as a NOTE if you prefer!